osc-server/src/main/java/org/osc/core/broker/service/securitygroup/BindSecurityGroupService.java

```diff
@@ -368,6 +357,13 @@ private VirtualSystem validateAndLoadVirtualSystem(EntityManager em, Long vsId)
 	private ServiceFunctionChain validateAndLoadSfcBind(EntityManager em, BindSecurityGroupRequest request) throws Exception {
 
     	ServiceFunctionChain sfc = null;
+        if (!request.isBindSfc()) {
+            //user is trying to configure binding SFC with either UI which currently not supported or APIs that are not
+            //valid for SFC binding, so throw an exception.
+            throw new VmidcBrokerValidationException(String.format(
+                    "Unsupported Interface or API to bind/unbind Security Group %s to Virtual System with a Service Function Chain",
```

Comment:
since this is called via the UI as well. We should just say "Binding individual Virtual Systems/Services is not supported for the Security Group %s. Please bind to a Service Function Chain"